python/pyarrow/tests/test_ipc.py

```diff
@@ -136,6 +140,22 @@ def stream_fixture():
     return StreamFormatFixture()
 
 
+@pytest.fixture(params=[
+    FileFormatFixture(),
+    StreamFormatFixture()
+    # pytest.param(
+    #     pytest.lazy_fixture('file_fixture'),
+    #     id='File Format'
+    # ),
+    # pytest.param(
+    #     pytest.lazy_fixture('stream_fixture'),
+    #     id='Stream Format'
+    # )
```

Comment:
This is probably redundant?

Reply:
Good catch thanks :).  I had thought this was causing problems at one point but it wasn't so I needed to uncomment and use this instead as it has nicer names.